packages/xo-web/src/common/intl/messages.js

```diff
@@ -767,6 +767,11 @@ const messages = {
   srsForget: 'Forget SRs',
   srRemoveButton: 'Remove this SR',
   srNoVdis: 'No VDIs in this storage',
+
+  // ----- SR disks tab -----
+  multipleActiveVdis: ' and {nVdis} more',
```

Comment:
Add the first VDI as a variable as well since the sentence syntax might be different in other languages.